grafana-dashboard.tf

```diff
@@ -0,0 +1,36 @@
+locals {
+  graph_id_base = 100
+  apiary_data_buckets = 2
+}
+
+data "template_file" "graphs" {
```

Comment:
datalake has lot of resources, can we `grafana_graphs` for this name?  so that it is clear in terraform plan output.